### PR TITLE
add project for olaph

### PR DESCRIPTION
**Pull request by loafyloaf**

### CODE REVIEW

File: create-cloud-resources.yml [hunk 1]
```diff
@@ -35,6 +35,9 @@
         - username: rattboi
           email: bradon.kanyid@ibm.com
           isadmin: False
+        - username: olaph
```
Comment (mlangbehn) [inline create-cloud-resources.yml]: Please move this section up to above rattboi so we can keep our alphabetical ordering.

Comment (loafyloaf) [reply create-cloud-resources.yml]: ah yes, of course

**Review by rattboi — APPROVED** [review]
:+1:

**Comment by jasontclark** [comment]
Shall we squash and merge this one?

**Pull request merged**